qemu/tests/cfg/block_hotplug.cfg

```diff
@@ -67,38 +67,27 @@
             only one_pci
             sub_type_after_plug = block_resize
             blk_extra_params_stg0 += ",serial=TARGET_DISK0"
-            block_size_cmd = "fdisk -l | grep {0}"
-            block_size_pattern = ",\s+(\d+\s+b)ytes"
             disk_change_ratio = "1.5 0.5"
             repeat_times = 1
+            Linux:
+                block_size_cmd = "fdisk -l | grep {0}"
```

Comment:
block size is smaller than 2TB suggest to use fdisk, if the size is larger than 2TB suggest to use parted.

Reply:
xuwei, thanks for your point, get a disk size was automated on avocado-vt, so I will delete them from cfg file.